src/FSharp.Control.TaskSeq.Test/TestUtils.fs

```diff
@@ -316,10 +318,10 @@ module TestUtils =
         let getEmptyVariant variant : IAsyncEnumerable<int> =
             match variant with
             | EmptyVariant.CallEmpty -> TaskSeq.empty
-            | EmptyVariant.Do -> taskSeq { do ignore () }
+            | EmptyVariant.Do -> taskSeq { do () }
```

Comment:
I like to be explicit here
```suggestion
            | EmptyVariant.Do -> taskSeq { do ignore () }
```

Reply:
Fair enough, but that's triply explicit over `()` or `do ()`, which is why Rider suggests it as a removal